Directory.Build.props

```diff
@@ -25,4 +25,11 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Npgsql.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
+
+  <!-- Reference .NET Framework reference assemblies, allows building on environments without .NET Framework installed
+       (e.g. Linux) -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.1" PrivateAssets="All" />
```

Comment:
The bootstrapping of this package is wrapped in the same target framework condition, so you can omit the condition if you wish.

Reply:
You mean that if I target it from a netcoreapp3.0 project it gets ignored?

Reply:
Yep, looks that way, thanks!